distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -0,0 +1,367 @@
+
+[IMPORTANT]
+====
+Although ${branding} can be installed by any user, it is recommended for security reasons to have a non-`root` user execute the ${branding} installation.
+====
+
+=== ${branding} Installation Prerequisites
+
+These are the system/environment requirements to configure _prior_ to an installation.
+
+=== Installing With the ${branding} Distribution Zip
+
+To install the ${branding} distribution zip, perform the following:
+
+* After the prerequisites have been met, as a root user if for *NIX, change the current directory to the desired install location.
+This will be referred to as `<INSTALL_DIRECTORY>`.
+
+.*NIX Tip
+[TIP]
+It is recommended that the root user create a new install directory that can be owned by a non-root user (e.g., ${branding-lowercase}-user).
+The non-root user (e.g., ${branding-lowercase}-user) can now be used for the remaining installation instructions.
+
+.Example: Create a Directory and Switch User on *NIX
+----
+mkdir new_installation
+chown ${branding-lowercase}-user:${branding-lowercase}-group new_installation
+
+su - ${branding-lowercase}-user
+----
+
+* Change the current directory to location of zip file (${branding-lowercase}-X.Y.zip).
+
+.Example: Where the Zip File may be Located in *NIX
+----
+cd /home/user/cdrom
+----
+
+.Windows (Example assumes ${branding} has been downloaded to the D drive)
+----
+cd D:\
+----
+* Copy ${branding-lowercase}-X.Y.zip to <INSTALL_DIRECTORY>.
+
+.*NIX
+----
+cp ${branding-lowercase}-X.Y.zip <INSTALL_DIRECTORY>
+----
+
+.Windows
+----
+copy ${branding-lowercase}-X.Y.zip <INSTALL_DIRECTORY>
+----
+
+* Change the current directory to the desired install location.
+
+.*NIX or Windows
+----
+cd <INSTALL_DIRECTORY>
+----
+
+* The ${branding} zip is now located within the `<INSTALL_DIRECTORY>`. Unzip ${branding-lowercase}-X.Y.zip.
+
+.*NIX
+----
+unzip ${branding-lowercase}-X.Y.zip
+----
+
+.Example: Use Java to Unzip in Windows
```

Comment:
Not sure why we say `Example` here. Could we just write `Windows (Use Java to Unzip)` or something similar?